### ISSUE TITLE
Technological resources metric or ratings for resource intensive lessons

### ISSUE THREAD

**Issue by JoshuaGOB** [programminghistorian/jekyll#1712]
I've been in communication with @jenniferisasi over the lesson on ImagePlot #254 and had a number of suggestions from one of the reviewers regarding a disclaimer or warning about the technological resources required to complete that lesson. This concern has come up in other spaces and I wanted to see if there are ways of addressing it beyond the initial review of a proposed lesson or text included within the lesson itself. 

Basically, resource intensive lessons shouldn't be excluded if they are relevant to what we do here but I think it might be useful for reviewers and the people using the lesson to know this going in. Something similar to how we have a difficulty rating might be an option. 

Thoughts? Is this more related to our Education manager @amsichani?

**Comment by mdlincoln** [comment]
I agree it makes sense to include an alert at the start noting that it might require relatively substantial computing resources. But you must translate it into real terms (e.g. you will probably need at least 8GB of ram to finish this lesson).

Because this doesn't apply to every single lesson I don't think it warrants a metadata tag like we have for general lesson difficulty. So I'm going to remove the "technical" label since it shouldn't require editing site templates to add this to a lesson.

**Comment by acrymble** [comment]
I would suggest this is an editorial matter, and should be up to the editorial boards to implement (I think it's a good idea. Thanks @JoshuaGOB ).

**Comment by rivaquiroga** [comment]
This is a very good idea @JoshuaGOB. Maybe we can use the alert formatting to warn about technological resources needed to complete the lesson (ram, space, admin privileges, etc.)

**Comment by acrymble** [comment]
This ticket does not have an owner @mariajoafana so no one is responsible for it. Can we arrange for someone to either take the idea forward, or to close it if we don't have capacity?

**Comment by mariajoafana** [comment]
I will bring this up in the next Project Team meeting. This could be stated in the editorial guidelines as something like this: 

If the lesson requires substantial computing resources we recommend adding an alert warning explaining the specific resources needed. (e.g. you will need at least 8GB of ram to finish this lesson).

**Comment by jenniferisasi** [comment]
I see we covered this on the July 28th meeting but, has anyone taken the lead on adding this on the guidelines?

**Comment by rivaquiroga** [comment]
I'll be working in the next few days on adding this to the guidelines

**Comment by rivaquiroga** [comment]
Hi everyone,

Reading our Author Guidelines it seems that a good place to add a note about resource intensive lessons could be the section ["Write for a Global Audience"](https://programminghistorian.org/en/author-guidelines#write-for-a-global-audience
), as it's related to taking steps towards writing a lesson that is accessible for as many people as possible. We can add somthing like this:



> * **Computing resources**: if your lesson requires relatively substantial computing resources, include an alert warning after the Table of Contents to inform the readers. Please be specific and translate the requirements into real terms (e.g. "You need at least 8GB of RAM to finish this lesson", "This lesson uses large files (up to 2GB)", etc. )



Question: Do we want the alert to be in a specific place (e.g., after the table of contents, in the Setup/Preparation/Prerequisites section, etc.)?

**Comment by jenniferisasi** [comment]
Thanks @rivaquiroga for working on this. I agree that the guidelines for global audiences is a good place to put a note. Because there are already a few banners before the ToC, I'd say it goes after it? It wouldn't be too many lessons anyways, but the top is already crowded.

**Comment by rivaquiroga** [comment]
After the ToC, sounds great. I just updated the note to reflect the change. 
@svmelton, @spapastamkou, @DanielAlvesLABDH, what do you think?  If you are ok with this we can move forward and open a PR.

**Comment by DanielAlvesLABDH** [comment]
Thanks for the note @rivaquiroga, for me it's fine

**Comment by svmelton** [comment]
Yes, this looks great!

**Comment by acrymble** [comment]
@rivaquiroga I'm going to close this ticket because it's been more than a year, and it looks like it will resolve when you merge your pull request.
